2.13.16.md

```diff
@@ -0,0 +1,68 @@
+# Scala 2.13.16 (DRAFT)
+
+The Scala team at Akka (formerly Lightbend) is pleased to announce Scala 2.13.16.
+
+The following changes are highlights of this release:
+
+### Breaking change
+
+* On the empty string, `.tail` and `.init` now throw (instead of returning the empty string) ([#10851](https://github.com/scala/scala/pull/10851) by [@sh0hei](https://github.com/sh0hei))
+    * This is a bugfix, but be aware that existing code might, perhaps inadvertently, be relying on the old behavior
+
+### Compatibility
+
+* Support Scala 3.6 in TASTy reader ([#10893](https://github.com/scala/scala/pull/10893))
+* Support JDK 24 in optimizer (via ASM upgrade) ([#10888](https://github.com/scala/scala/pull/10888) by [@Philippus](https://github.com/Philippus))
+
+### REPL
+
+* REPL: JLine 3.27.1 (was 3.26.3) and on Windows use JNA not JNI ([#10899](https://github.com/scala/scala/pull/10899), [#10898](https://github.com/scala/scala/pull/10898), [#10882](https://github.com/scala/scala/pull/10882))
+    * These upgrades hopefully won't be noticed, but, REPL users on Windows take note
+
+### Align with Scala 3
+
+* Under `-Xsource:3`, deprecate infix named args ([#10857](https://github.com/scala/scala/pull/10857) by [@som-snytt](https://github.com/som-snytt))
+
+### Errors and warnings
+
+* Fix 2.13.15-only false positives with `-Wunused:patvars` ([#10870](https://github.com/scala/scala/pull/10870) by [@som-snytt](https://github.com/som-snytt))
+
+### Collections
+
+* Do not use `rangeHash` when `rangeDiff` is 0 ([#10912](https://github.com/scala/scala/pull/10912) by [@Friendseeker](https://github.com/Friendseeker))
+    * This may affect fragile code relying on ordering of hash-based collections
+* Deprecate `collection.mutable.AnyRefMap` ([#10862](https://github.com/scala/scala/pull/10862) by [@wangyum](https://github.com/wangyum))
+    * It no longer has any performance advantage over `collection.mutable.HashMap`
```

Comment:
"no longer" ... since when?

What is the recommendation for cross-compiling code, where Scala 2.12 performance still matters? (in my case, a lot, since it's linker code loaded through an sbt plugin)

Should we move to `HashMap` anyway, or should we ignore the deprecation with a compiler flag instead?

Reply:
since 2.13.0, as per https://github.com/scala/scala/releases/tag/v2.13.0 — I've pushed an edit to say so

for cross-compiling, I would suggest suppressing the deprecation warning, yes. (I mean if you really wanted you could use version-specific sources...)